core/store/src/lib.rs

```diff
@@ -45,6 +47,30 @@ pub mod migrations;
 pub mod test_utils;
 mod trie;
 
+/// Describe database settings for column.
+///
+/// If needed we can add more settings.
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct DbColSettings {
+    /// Lru cache for column
+    #[serde(skip_serializing_if = "Option::is_none")]
+    new_lru_cache: Option<u64>,
```

Comment:
Let's see how it looks like in the config file:

```json
"db_settings": {
  "col_settings": {
    "ColState": {
      "new_lru_cache": 10
     }
  },
  "default_col_settings": {
    "new_lru_cache": 10
  }
}
```

I know that we already have a config file that is hard to reason about, but this does not help us...